source/about/zero-trust.rst

```diff
@@ -0,0 +1,127 @@
+
+Security policies and tokens
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enhance security with tailored authentication tools:  
+
+- `Personal Access Tokens <https://developers.mattermost.com/integrate/reference/personal-access-token/>`_: Enables secure API access with identity verification aligned to least privilege and Zero Trust requirements.
+
+Continuous monitoring
+----------------------
+
+To align with an always verify philosophy, Mattermost offers tools for monitoring activity, identifying suspicious behavior, session management, and real-time incident response. Audit trails and performance monitoring ensure proactive detection of potential issues or breaches, delivering visibility into the activity across the platform. Session management strengthens the protection against session hijacking or unauthorized persistent access.  
+
+- `Audit Logging <https://docs.mattermost.com/manage/logging.html>`_: Tracks detailed activity logs for monitoring and identifying anomalous behaviors, supporting detection capabilities.
+- `SIEM Integrations <https://developers.mattermost.com/integrate/webhooks/>`_: Streamlines monitoring within existing security systems to detect and respond to threats consistently with Zero Trust principles.
+- `Performance Monitoring <https://docs.mattermost.com/scale/deploy-prometheus-grafana-for-performance-monitoring.html>`_: Protects against potential threats by analyzing system and user behaviors via proactive monitoring.
+
+Multi-factor authentication (MFA)
+----------------------------------
+
+Adding an extra layer of protection beyond passwords is essential for Zero Trust. Mattermost supports MFA to strengthen authentication practices. MFA ensures that unauthorized users are denied access even if passwords are compromised, reducing the risk of account breaches.  
+
+Alternatively, often enforced through the identity provider (IDP).  
+
+- `MFA <https://docs.mattermost.com/onboard/multi-factor-authentication.html>`_: Enhances user identity verification by requiring multiple factors for authentication in adherence to Zero Trust standards.
+
+Encryption
+----------
+
+Encryption protects both data at rest and data in transit, ensuring end-to-end security for sensitive communications. Encryption mitigates the risk of data theft in both storage and transfer, while granular permissions limit access to sensitive files and data to only authorized users.  
+
+- `Database Encryption <https://docs.mattermost.com/deploy/encryption-options.html#database>`_: Protects user and organizational data at rest, safeguarding sensitive information from unauthorized access.
+- `TLS Encryption <https://docs.mattermost.com/deploy/encryption-options.html#encryption-in-transit>`_: Secures data in transit by encrypting communications.
+
+File storage encryption
```

Comment:
Let's delete this section and consolidate:

1. Should we move this to Encryption?
2. Let's move this to Administartive Controls?
3. Same as 2